src/scripts/main.js

```diff
@@ -1,3 +1,172 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const employeeTable = document.querySelector('table');
+  let currentSort = { column: null, order: 'asc' };
+  let selectedRow = null;
+
+  const headers = document.querySelectorAll('th');
+
+  headers.forEach((header, index) => {
+    header.addEventListener('click', () => {
+      const order =
+        currentSort.column === index && currentSort.order === 'asc'
```

Comment:
This line is a bit confusing and hard to read. It would be better if you split this into multiple lines or even separate if-else statements for better readability.